addons/account_peppol/tests/test_peppol_messages.py

```diff
@@ -64,9 +64,10 @@ def setUpClass(cls):
             'partner_id': cls.env.company.partner_id.id,
         })
 
-    def create_move(self, partner):
+    def create_move(self, partner, company=None):
```

Comment:
Imo it would be cleaner to have **kwargs here to handle both company_id and partner_id without adding new parameters everytime we need to passe an additional value to the create but I understand it implies more refactoring.
Feel free to ignore ~~